librz/core/disasm.c

```diff
@@ -290,9 +290,9 @@ typedef struct {
 	// caches
 	char *_tabsbuf;
 	int _tabsoff;
-	bool dwarfFile;
-	bool dwarfAbspath;
-	bool dwarfShowLines;
+	bool asm_debuginfo_file;
```

Comment:
maybe remove `asm_` prefix? It's a structure of the disasm state after all.

Reply:
Also maybe make it a substructure instead? So that it could be modified/extended easily when necessary?